### ISSUE TITLE
Any way to shorten a warm-up time?

### ISSUE THREAD

**Issue by SuperMaximus1984** [neonbjb/tortoise-tts#632]
Is there any way to keep the model preloaded so that a long pause before "Generating autoregressive samples.." could be shortened?

```
/home/max/anaconda3/envs/tortoise/lib/python3.9/site-packages/torch/nn/utils/weight_norm.py:30: UserWarning: torch.nn.utils.weight_norm is deprecated in favor of torch.nn.utils.parametrizations.weight_norm.
  warnings.warn("torch.nn.utils.weight_norm is deprecated in favor of torch.nn.utils.parametrizations.weight_norm.")
Generating autoregressive samples..
100%|████████████████████████████████████████████| 1/1 [00:04<00:00,  4.90s/it]
Computing best candidates using CLVP
100%|████████████████████████████████████████████| 1/1 [00:00<00:00,  6.10it/s]
Transforming autoregressive outputs into audio..
100%|██████████████████████████████████████████| 30/30 [00:01<00:00, 18.86it/s]
100%|██████████████████████████████████████████| 30/30 [00:01<00:00, 23.37it/s]
100%|██████████████████████████████████████████| 30/30 [00:01<00:00, 22.74it/s]

real    0m44.433s
user    0m47.928s
sys     0m9.528s
```

**Comment by manmay-nakhashi** [comment]
now this is much faster with api_fast , 
generating audio..
Time taken to generate the audio:  6.466272592544556 seconds
RTF:  0.4754612200400409
Generating autoregressive samples..
generating audio..
Time taken to generate the audio:  3.695239782333374 seconds
RTF:  0.2434495640152873
Generating autoregressive samples..
generating audio..
Time taken to generate the audio:  3.573507070541382 seconds
RTF:  0.25076069450842403
Generating autoregressive samples..
generating audio..
Time taken to generate the audio:  3.7674925327301025 seconds
RTF:  0.2444307439054997
Generating autoregressive samples..
generating audio..
Time taken to generate the audio:  3.2888433933258057 seconds
RTF:  0.26151744539804433
Generating autoregressive samples..
generating audio..
Time taken to generate the audio:  3.148411750793457 seconds
RTF:  0.2702963384953174
Generating autoregressive samples..
generating audio..
Time taken to generate the audio:  4.243706226348877 seconds
RTF:  0.2688158504866265
Generating autoregressive samples..
generating audio..
Time taken to generate the audio:  1.7666492462158203 seconds
RTF:  0.2931387023588198
Generating autoregressive samples..
generating audio..
Time taken to generate the audio:  2.6709401607513428 seconds
RTF:  0.2819827027820252
Generating autoregressive samples..
generating audio..
Time taken to generate the audio:  2.619523763656616 seconds
RTF:  0.26491947447983577
Generating autoregressive samples..
generating audio..
Time taken to generate the audio:  3.833838939666748 seconds
RTF:  0.2735330293712006

**Comment by SuperMaximus1984** [comment]
@manmay-nakhashi Could you please elaborate it? How to use api_fast?
Thanks!

**Comment by manmay-nakhashi** [comment]
Use read_fast to know how it works
